utopian-io/v2.utopian.io
# Comment vote

### Verify that
Git checklist

Front end checklist

API checklist

Tests and comments checklist

### **Test plan**

* Go to http://localhost:8080/en/articles/adriel/another-test-1547586974631
* Verify that some of the comments have upvotes
* Try to upvote them and un-upvote them
